Times jobs

tanishkunigiri opened the pull request:
## Description

> The changes made in this pull request should be briefly described.
Added TImesJobs which gives information regarding the companies hiring for a particular Job role. Uses Beautiful Soup for getting the contents from the webpage of TimesJobs.
## Resolves: [Issue Number]
379
## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

> Uploading a screenshot illustrating the approach you developed for validation is required.
![Python](https://github.com/Clueless-Community/scrape-up/assets/106328619/2cd9b273-d792-4f73-afd5-5f01af129dbd)

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.
Here we need not print any the result. The result is automatically shown in the terminal or notebook.
---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

nikhil25803 reviewed (changes requested):
Requested some changes @tanishkunigiri |

tanishkunigiri commented:
Shall i request a new PR with the following changes that u have requested for?

tanishkunigiri commented:
I have also changed a few lines of code including changing docstrinig, returning data in JSON format and asking for the Job role required at the time of instantiation. I also added few additional features for checking if the input data is invalid or valid

nikhil25803 commented:
No @tanishkunigiri | make the changes in this PR only. 
Have you already pushed the new changes?

tanishkunigiri commented on src/scrape_up/TImesJobs/timesjobs_scraper.py:
```diff
@@ -0,0 +1,61 @@
+        Application Links: The link which directly takes you to the Web-page where you can fill-in the details\n
+        
+        '''
+        Job_role = input('Enter your desired role: ')
```
I have made the required changes for this in TImesJobs, pls go through it

tanishkunigiri commented on src/scrape_up/TImesJobs/timesjobs_scraper.py:
```diff
@@ -0,0 +1,61 @@
+        Location: The location at which the company is located\n
+        Experience: The experience of the applicants required for the post\n
+        Days: The number of days before which this job was posted on this webiste\n
+        Application Links: The link which directly takes you to the Web-page where you can fill-in the details\n
```
I have made the required changes for this in TImesJobs, pls go through it

tanishkunigiri commented on src/scrape_up/TImesJobs/timesjobs_scraper.py:
```diff
@@ -0,0 +1,61 @@
+                
+                
+                print(f'Company: {company}\nLocation: {location}\nExperience: {experience}\nPosted: {days} ago')
+                print(f'Apply here: {href_value}\n')
```
I have made the required changes for this in TImesJobs, pls go through it
_

nikhil25803 commented:
Great work @tanishkunigiri !!
Do not forget to
+ Share your contribution on socials(twitter mainly), tag us and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅
